### DISCUSSION TITLE
centeredSlides not working with slidesOffsetBefore

### DISCUSSION THREAD

**Discussion by sanderschnydrig** [nolimits4web/swiper#7511] [Q&A]
When using centeredSlides and centeredSlidesBounds and slidesOffsetBefore together the slides are not centered in the middle of the slider anymore. I want to have the first and last slides hug the edge of the container but with the offset, so they are not at the edge of the screen (the container extends to the edge of the screen so the slides that are not fully visible will extend to the edge of the screen but not the active slides).

https://stackblitz.com/edit/stackblitz-starters-1sz8yp?file=src%2Fapp%2Fapp.component.ts

My expected behavior would be that the offset only counts for the first and last slides, but it seems I need to implement a workaround with the current version of swiper.js.

**Comment by lamlamla** [comment]
Facing same issuse here. Does anyone have a workaround ?
